collector/container/konflux.Dockerfile

```diff
@@ -0,0 +1,196 @@
+        libubsan \
+        libcap-ng-devel \
+        libcurl-devel \
+        libtool \
+        libuuid-devel \
+        make \
+        openssh-server \
+        openssl-devel \
+        patchutils \
+        passwd \
+        pkgconfig \
+        rsync \
+        tar \
+        unzip \
+        valgrind \
+        wget \
+        which \
+        bpftool \
+    && dnf clean all
+
+ARG BUILD_DIR
+ARG SRC_ROOT_DIR=${BUILD_DIR}
+ARG CMAKE_BUILD_DIR
+# TODO(ROX-20240): CMAKE_BUILD_TYPE should probably not be Release for PR, normal branch builds
+ARG CMAKE_BUILD_TYPE=Release
+# Appends an argument to the driver download URL that is used for filtering alerts on missing kernels.
+# TODO(ROX-20240): This needs to be true on PRs only.
+ARG COLLECTOR_APPEND_CID=false
+ARG USE_VALGRIND=false
+ARG ADDRESS_SANITIZER=false
+ARG TRACE_SINSP_EVENTS=false
+
+WORKDIR ${BUILD_DIR}
+
+RUN mkdir kernel-modules \
+    && cp -a /builder builder \
+    && cp -a /collector collector \
+    && cp -a /falcosecurity-libs falcosecurity-libs \
+    && cp -a /builder/third_party third_party \
+    && cp -a /kernel-modules/MODULE_VERSION kernel-modules/MODULE_VERSION \
+    && cp -a /CMakeLists.txt CMakeLists.txt
+
+# WITH_RHEL_RPMS controls for dependency installation, ie if they were already installed as RPMs.
+# Setting the value to empty will cause dependencies to be downloaded from repositories or accessed in submodules and compiled.
+# TODO(ROX-20651): Set ENV WITH_RHEL_RPMS=true when RHEL RPMs can be installed to enable hermetic builds.
+# ENV WITH_RHEL_RPMS=true
+
+# Build with gperftools (DISABLE_PROFILING=OFF) only for supported
+# architectures, at the moment x86_64 only
+RUN ./builder/install/install-dependencies.sh && \
+    if [[ "$(uname -m)" == "x86_64" ]];   \
+        then DISABLE_PROFILING="OFF";   \
+        else DISABLE_PROFILING="ON";    \
+    fi ; \
+    cmake -S ${SRC_ROOT_DIR} -B ${CMAKE_BUILD_DIR} \
+           -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
+           -DDISABLE_PROFILING=${DISABLE_PROFILING} \
+           -DCOLLECTOR_APPEND_CID=${COLLECTOR_APPEND_CID} \
+           -DUSE_VALGRIND=${USE_VALGRIND} \
+           -DADDRESS_SANITIZER=${ADDRESS_SANITIZER} \
+           -DTRACE_SINSP_EVENTS=${TRACE_SINSP_EVENTS} && \
+    cmake --build ${CMAKE_BUILD_DIR} --target all -- -j "${NPROCS:-2}" && \
+    ctest -V --test-dir ${CMAKE_BUILD_DIR} && \
+    strip -v --strip-unneeded "${CMAKE_BUILD_DIR}/collector/collector"
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS support-packages-downloader
+
+WORKDIR /staging
+
+COPY kernel-modules/MODULE_VERSION kernel-modules/MODULE_VERSION
+COPY collector/container/scripts/download-support-package.sh download-support-package.sh
+
+RUN ./download-support-package.sh
+
+# Do NOT use follow_tag here, as we do not need or want collector to be rebuilt
+# with each drivers build (which may become very frequent)
+FROM brew.registry.redhat.io/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
```

Comment:
The comment is valid in CPaaS world where `follow_tag` is a thing. However, Konflux does not know it. Let's reword it a bit while having the main message the same.

```suggestion
# 0.1.0 is a floating tag and it's used intentionally to pick up the most recent downstream drivers build without
# having to routinely and frequently bump tags here.
FROM brew.registry.redhat.io/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
```